### PR TITLE
Replace SmallRyeTracingCDIInterceptor with OpenTracingInterceptor

### PR DESCRIPTION
**Pull request by jpkrohling**
Signed-off-by: Juraci Paixão Kröhling <juraci@kroehling.de>

### CODE REVIEW

**Comment by pavolloffay** [comment]
Could you please update root readme?

**Comment by jpkrohling** [comment]
Updated

**Comment by jpkrohling** [comment]
I just updated this PR to use the latest version of the interceptors repo, incorporating the latest change related to the logging of errors in the span. @pavolloffay could you please review?

File: tck/src/test/java/io/smallrye/opentracing/OpenTracingCDIExtension.java [hunk 1]
```diff
@@ -16,6 +16,5 @@
   public void observeBeforeBeanDiscovery(@Observes BeforeBeanDiscovery bbd, BeanManager manager) {
     logger.info("Registering Tracer CDI producer");
     bbd.addAnnotatedType(manager.createAnnotatedType(TracerProducer.class));
-    bbd.addAnnotatedType(manager.createAnnotatedType(SmallRyeTracingCDIInterceptor.class));
```
[the remark this answers is not in the document]
Comment (jpkrohling) [reply tck/src/test/java/io/smallrye/opentracing/OpenTracingCDIExtension.java]: Yes, the archive has a `beans.xml`, which is enough to have it found.

**Pull request merged**